metrics/helpers/lp.py

```diff
@@ -123,6 +123,57 @@ def get_team_daily_triage_count(team, distro, blacklist=None):
     return len(results)
 
 
+def get_team_unassigned_bugs(team, distro):
```

Comment:
To clarify my previous comment I think this should be renamed "get_team_subscribed_unassigned_bugs" given that searchTasks has additional parameters like bug_supervisor and structural_subscriber and those types of filters may be used in the future.